pytorch_cnn_trainer/engine.py

```diff
@@ -333,9 +333,19 @@ def fit(
                 fp16_scaler=scaler,
             )
 
+        loss , top1_acc , top5_acc = [train_metrics[i] for i in ('loss','top1','top5')]
+        history = {"train" : {"top1_acc" : top1_acc , "top5_acc" : top5_acc, "loss" : loss},"val" : { }}
+
+
         print()
         print(f"Validating Epoch = {epoch}")
         valid_metrics = val_step(model, valid_loader, criterion, device, num_batches, log_interval)
+        
+        loss_v , top1_acc_v , top5_acc_v = [train_metrics[i] for i in ('loss','top1','top5')]
```

Comment:
ohh sorry I haven't changed train_metrics here , I just did copy paste so forgot

Reply:
Looks better now, this is cleaner right?

Can you add code to plot this using `matplotlib`. This can go in seperate file called `plotters.py` (name suggestions needed !)